docs/web3.contract.rst

```diff
@@ -591,6 +591,20 @@ size, web3 will invalidate the value. For example, if an abi specifies a type of
      - Needs to have exactly 4 bytes
 
 
+In cases where functions are overloaded and use arguments of similar types, a function
```

Comment:
This is a bit in the middle of the `strict_bytes_type_checking` flag explainer. I think we can find a better spot for this so as to not confuse users that this is somehow related to the flag dynamics.

Reply:
Never mind, it got moved appropriately 😅

Reply:
Oh okay. I'll move this to the other section where I just revamped the ContractFunctions docs